### PR TITLE
Next js doc changes

### PR DESCRIPTION
**Pull request by jay-khatri**

### CODE REVIEW

File: general-docs/2_getting-started/fullstack-frameworks/next-js/1_overview.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ updatedAt: 2022-10-18T22:40:13.000Z
...
 The Highlight Next.js SDK adds additional features to Highlight, including:
 
--   server-side error monitoring and linking to Highlight sessions: [Highlight()](/getting-started/nextjs-sdk/highlight)
+--   server-side error monitoring and linking to Highlight sessions: [Highlight()](/docs/sdk/nextjs#Highlight)
 
--   automatic configuration of source map uploads: [withHighlightConfig()](/getting-started/nextjs-sdk/with-highlight-config)
+--   automatic configuration of source map uploads: [withHighlightConfig()](/docs/sdk/nextjs#withHighlightConfig)
 
--   automatic proxying for Highlight requests using Next.js rewrites: [withHighlightConfig()](/getting-started/nextjs-sdk/with-highlight-config)
+--   automatic proxying for Highlight requests using Next.js rewrites: [withHighlightConfig()](/docs/sdk/nextjs#withHighlightConfig)
```
Comment (Vadman97) [inline general-docs/2_getting-started/fullstack-frameworks/next-js/1_overview.md:14]: the double `--` makes the frontend not render this like a normal bulleted list

**Review by Vadman97 — CHANGES_REQUESTED** [review]
need to update `middleware.ts` to the new next path
image

File: general-docs/2_getting-started/fullstack-frameworks/next-js/1_overview.md [hunk 2]
```diff
@@ -7,11 +7,11 @@ updatedAt: 2022-10-18T22:40:13.000Z
 
 The Highlight Next.js SDK adds additional features to Highlight, including:
 
--   server-side error monitoring and linking to Highlight sessions: [Highlight()](/getting-started/nextjs-sdk/highlight)
+--   server-side error monitoring and linking to Highlight sessions: [Highlight()](/docs/sdk/nextjs#Highlight)
```
[the remark this answers is not in the document]
Comment (jay-khatri) [reply general-docs/2_getting-started/fullstack-frameworks/next-js/1_overview.md]: why do the links in the `hint` in `angular.md` not work without the `/docs` path then?

**Review by Vadman97 — APPROVED** [review]

File: general-docs/2_getting-started/fullstack-frameworks/next-js/1_overview.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Vadman97) [reply general-docs/2_getting-started/fullstack-frameworks/next-js/1_overview.md]: looks like the `hint` [callout processing](https://github.com/highlight-run/highlight-landing/blob/main/components/Docs/Callout/Callout.tsx) is different and uses the regular `ReactMarkdown` processing. that's why the links actually reload the tab as well which is not great. filing a ticket to fix both.

**Pull request merged**